pyccl/core.py

```diff
@@ -390,6 +388,10 @@ def _build_config(
                              "method. Available options are: %s"
                              % (emulator_neutrinos,
                                 emulator_neutrinos_types.keys()))
+        if (matter_power_spectrum == "camb"
+                and transfer_function != "boltzmann_camb"):
+            raise CCLError("To compute the matter power spectrum with CAMB "
+                           "the transfer function should be 'boltzmann_camb'.")
```

Comment:
Can you specify that this is the non-linear matter power spectrum?

Reply:
Done.